pythonscript/godot/_hazmat/internal.pyx

```diff
@@ -1,20 +1,62 @@
 cdef bint __pythonscript_verbose = False
-
+import threading
 
 # /!\ This dict is strictly private /!\
 # It contains class objects that are referenced
 # from Godot without refcounting, so droping an
 # item from there will likely cause a segfault
 cdef dict __exposed_classes_per_module = {}
+__all_loaded_classes = {}
+__exposed_classes_lock = threading.Lock()
 
-
+import sys
 cdef object get_exposed_class(str module_name):
-    return __exposed_classes_per_module.get(module_name)
+    sys.stdout.flush()
```

Comment:
Why a flush here ? (this would definitely requires a comment !)

Reply:
my mistake! this is debug code that I forgot to remove.